index.js

```diff
@@ -64,7 +64,10 @@ function formatAttrs(attributes, opts) {
     if (!value && booleanAttributes[key]) {
       output += key;
     } else {
-      output += key + '="' + (opts.decodeEntities ? entities.encodeXML(value) : value) + '"';
+      output += key;
+      if (value || opts.xmlMode) {
```

Comment:
`value` is a string. I don't really like this approach – and the use-cases rely on faulty behavior already (ie. attribute names with invalid characters), which I would rather not continue to support.